examples/tls/tls_common.h

```diff
@@ -311,6 +322,240 @@ static inline void CloseAndCleanupSocket(SockIoCbCtx* sockIoCtx)
         sockIoCtx->listenFd = -1;
     }
 }
+#else /* USE_WOLFSSL_IO */
```

Comment:
Can we combine this section with the one above? A lot of duplicate code there. I'd prefer we pull in the `USE_WINDOWS_API` logic and socket mappings here directly. I have code for this.

Reply:
Will do.